src/test/java/millom/sandbox/service/MessageServiceImplTest.java

```diff
@@ -15,12 +17,18 @@ void setUp() {
   }
 
   @Test
-  void TestConvertCuriositySol() {
+  void convertCuriositySolTest1() throws DateFormatIncorrectException {
```

Comment:
This and the next one are good candidates for a parametrised test: https://www.baeldung.com/parameterized-tests-junit-5

Reply:
added. dependency for parameterised test.
`testImplementation 'org.junit.jupiter:junit-jupiter-params:5.8.2'`

And changed the tests to single test.
```
@ParameterizedTest
  @CsvSource(value = {"2022-06-15:3504", "2022-06-27:3516"}, delimiter = ':')
  void ConvertCuriositySolTest(String input, int expected) throws DateFormatIncorrectException {
    assertEquals(expected, messageServiceImpl.convertCuriositySol(input));
  }
```